gradio/components.py

```diff
@@ -5549,7 +5690,7 @@ class Code(Changeable, IOComponent, SimpleSerializable):
 
     def __init__(
         self,
-        value: str | None = None,
+        value: str | Tuple[str] | None = None,
```

Comment:
I don't think this type hint is accurate

Reply:
Why not? This is the type if a filepath is passed in

Reply:
Sorry I read the diff incorrectly and thought this was getting added to Model3D as opposed to code lol